sdk/cosmos/azure-cosmos/test/test_vector_policy.py

```diff
@@ -0,0 +1,252 @@
+# The MIT License (MIT)
+# Copyright (c) Microsoft Corporation. All rights reserved.
+
+import unittest
+import uuid
+
+import pytest
+
+import azure.cosmos.exceptions as exceptions
+import test_config
+from azure.cosmos import CosmosClient, PartitionKey
+
+
+@pytest.mark.cosmosEmulator
+class TestAutoScale(unittest.TestCase):
+    client: CosmosClient = None
+    host = test_config.TestConfig.host
+    masterKey = test_config.TestConfig.masterKey
+    connectionPolicy = test_config.TestConfig.connectionPolicy
+
+    @classmethod
+    def setUpClass(cls):
+        if (cls.masterKey == '[YOUR_KEY_HERE]' or
+                cls.host == '[YOUR_ENDPOINT_HERE]'):
+            raise Exception(
+                "You must specify your Azure Cosmos account values for "
+                "'masterKey' and 'host' at the top of this class to run the "
+                "tests.")
+
+        cls.client = CosmosClient(cls.host, cls.masterKey)
+        cls.created_database = cls.client.get_database_client(test_config.TestConfig.TEST_DATABASE_ID)
+
```

Comment:
We would have to skip/ignore all the vector_policy test cases for now, as they require a new build of public emulator.